lib/players/human.rb

```diff
@@ -0,0 +1,10 @@
+module Players
+  class Human < Player
+
+    def move(board)
+      "Please enter 1-9: "
+      user_input = gets.strip
+      user_input
+    end
```

Comment:
We don't really need to create a `user_input` variable here.  Since we're returning the user's input, we could just leave the last line of the method as `gets.strip`, as shown below.  Also, don't forget to `puts` the prompt ;)
``` ruby
def move(board)
  puts "Please enter 1-9: "
  gets.strip
end
```